contribution-guide.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: ballerina-inner-page
+title: The Ballerina Contribution Guide
+permalink: /contribution-guide/
+---
+
+# The Ballerina Contribution Guide
+
+Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software. It is an open-source project under the Apache license and the work of hundreds of contributors.
+
+We appreciate your help!
+
+- [Getting engaged](#getting-engaged)
+- [Getting started](#getting-started)
+- [Obtaining the source code](#obtaining-the-source-code)
+- [Building the project](building-the-project)
+- [Setting up the developer environment](setting-up-the-developer-environment)
+    - [Setting up IntelliJ IDEA](#setting-up-intelliJ-idea)
+    - [Setting up Eclipse](#setting-up-eclipse)
+- [Contributing to Ballerina grammar](#contributing-to-ballerina-grammar)
+- [Submitting your contribution](#submitting-your-contribution)
+- [Accepting the Contributor License Agreement](#accepting-the-contributor-license-agreement)
+- [Proposing changes to Ballerina](#proposing-changes-to-ballerina)
+
+## Getting engaged
+
+- The [Ballerina-Dev Google Group](#https://groups.google.com/forum/#!forum/ballerina-dev) is the main Ballerina project discussion forum. Also, you can use [StackOverflow](#https://stackoverflow.com/questions/tagged/ballerina) for support, [Slack](#https://app.slack.com/client/T47EAEKB5/DF818FX9Q) for real-time communications.
+
+- **IMPORTANT:** Send an email to security@ballerina.io to report sensitive security-related issues. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
+
+- Submitting a bug is just as important as contributing to code. Click the **Issues** tab of the below GitHub repos, click **New Issue**, fill out all sections in the issue template, and submit the bug/issue report.
+
+    >**Tip:** If you are unsure whether you have found a bug, search existing issues in GitHub and raise it in the [Ballerina-Dev Google Group](#https://groups.google.com/forum/#!forum/ballerina-dev).
```

Comment:
```suggestion
    >**Tip:** If you are unsure whether you have found a bug, search existing issues on GitHub and raise it in the [Ballerina-Dev Google Group](#https://groups.google.com/forum/#!forum/ballerina-dev).
```

Reply:
You search in GitHub not on it.

Reply:
Please see https://help.github.com/en/github/searching-for-information-on-github/searching-on-github - these are the GitHub docs.

You search **in** a repository **on** GitHub.

Reply:
"In a repository on GitHub" is correct. But this is saying directly "in GitHub".

Reply:
Sorry, I don't agree. If it is "in GitHub" shouldn't the GitHub docs say "in multiple repositories **in** GitHub" instead of "in multiple repositories **on** GitHub"?

IMO, you do something on a site, not in it.

@pubudu91 suggested "search existing issues in the corresponding repo on GitHub", how about using that?

Reply:
+1 Changed to " "search existing issues in the corresponding repo on GitHub"".